### DISCUSSION TITLE
Date Format Configuration

### DISCUSSION THREAD

**Discussion by simon-online** [mirumee/saleor-dashboard#2125] [Q&A] [answered]
Is there an easy way to change the date format in saleor-dashboard?

For example under the product Availability panel it has the dates formatted as
since MM/DD/YYYY

and I'd like to be able to at least change it to
since DD/MM/YYYY

**Comment by timuric** [comment]
@simon-online not sure about settings, but we should definitely use users locale settings

**Comment by timuric** [comment, marked as answer]
PR https://github.com/saleor/saleor-dashboard/pull/2129

**Comment by timuric** [comment]
DD/MM formats can be confusing so I made a PR to use "1 Jan" format https://github.com/saleor/saleor-dashboard/pull/2130
